code/game/machinery/computer/dropship_weapons.dm

```diff
@@ -321,7 +321,7 @@
 							is_outside = TRUE
 						if(CEILING_GLASS)
 							is_outside = TRUE
-				if(!is_outside && !cavebreaker) //cavebreaker doesn't care
+				if(!is_outside && !(selected_equipment.ammo_equipped.ammo_ignore_caves)) //cavebreaker doesn't care
```

Comment:
Agreed.  Originally I was going back and forth on keeping it in case we ever just wanted a DS to just ignore caves/roofs, but I think this is more robust if we place that ability on the ammo types.